dev.bragefuglseth.Keypunch.json

```diff
@@ -0,0 +1,65 @@
+{
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin"
+  },
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "command": "keypunch",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland"
+  ],
+  "id": "dev.bragefuglseth.Keypunch",
+  "modules": [
+    {
+      "buildsystem": "meson",
+      "cleanup": [
+        "*"
+      ],
+      "name": "blueprint-compiler",
+      "sources": [
+        {
+          "tag": "v0.12.0",
+          "type": "git",
+          "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler"
+        }
+      ]
+    },
+    {
+      "builddir": true,
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dbuildtype=release",
+        "-Dsandboxed=true"
+      ],
+      "name": "keypunch",
+      "run-tests": true,
+      "sources": [
+        {
+          "commit": "655b21979a28100def963ca085de5302831d74b1",
+          "tag": "v1.0",
+          "type": "git",
+          "url": "https://github.com/bragefuglseth/keypunch.git"
+        },
+        "cargo-sources.json"
+      ]
+    }
+  ],
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ]
```

Comment:
Looks like they to the bottom?

Reply:
The formatter I used apparently alphabetized all the fields. I've fixed it now.